internal/imports/imports_linux.go

```diff
@@ -1,61 +0,0 @@
-// DO NOT EDIT - generated by github.com/edwarnicke/imports-gen
-package imports
```

Comment:
Why this is deleted?

Reply:
it wasn't manually deleted
result of "go generate", i think
should i restore it?